### DISCUSSION TITLE
How do I close Hamburger menu which is open, from the drop down list item inside the hamburger

### DISCUSSION THREAD

**Discussion by digish777** [vmware/clarity#5308] [Q&A] [answered]
https://github.com/vmware/clarity/issues/5294
You closed this issue with out checking with me.

I require drop down list in the left and in the right section. I cannot restrict the list only to left. I have total of 7 lists. Your opinion of one list will not work for me.

I have 5 lists in the left side. and 2 lists in the right.

help would be greatly appreciated.

And my question was, how to close the Hamburger menu, which I did not get the reply for.

**Comment by hippee-lee** [comment, marked as answer]
Hi @digish777 - this seems more of a `How does my app ...` question and not an issue. I'm going to move it into the discussions format. 

I believe what is needed is for when the app navigation is in responsive mode and the navigation is hidden/shown for responsive use cases. Please help me understand more if I am not right. 

The solution involves two things:

1. de-sugaring the `clr-dropdown-menu` so the app can use `clrIfOpenChange` events
2. getting a reference to the ClrHeader instance so the app can programetically close it 

Additionally I added some css that will move the drop down menu closer to the button that opens and closes it. 

Obviously my stackblitz is just a POC so you may need to handle some use cases or edge cases that are specific to the app but [this stackblitz](https://stackblitz.com/edit/gh-5308-dropdown-menu-location?file=src/app/app.component.html) should help you move forward.
